examples/triangle/main.rs

```diff
@@ -28,58 +34,51 @@ gfx_pipeline!(pipe {
     out: gfx::RenderTarget<ColorFormat> = "Target0",
 });
 
-struct App<R: gfx::Resources> {
-    pso: gfx::PipelineState<R, pipe::Meta>,
-    data: pipe::Data<R>,
-    slice: gfx::Slice<R>,
-}
-
-impl<R: gfx::Resources> gfx_app::Application<R> for App<R> {
-    fn new<F: gfx::Factory<R>>(mut factory: F, init: gfx_app::Init<R>) -> Self {
-        use gfx::traits::FactoryExt;
+fn triangle() -> [Vertex; 3] { [
```

Comment:
I'd prefer constants instead of global functions here. First, because constants are more restrictive. Second, because function names should start with a verb, and `triangle()` doesn't satisfy this rule, and `clear_color()` is misleading for it's not really clearing anything.